frontend/packages/ceph-storage-plugin/integration-tests/views/installFlow.view.ts

```diff
@@ -101,10 +124,16 @@ const OCP_44 = {
   searchInputOperators: $('input[placeholder="Filter by name..."]'),
 };
 
+const OCP_45 = {
+  independentModeButton: $('input[name="independent-mode"]'),
+};
```

Comment:
Isn't it OCP_46 ??

Reply:
No, it should be OCP_45 only as the latest one in OCP_46